skietbaan_fe/src/components/Register.js

```diff
@@ -0,0 +1,177 @@
+import React, { Component } from 'react';
+import { Container, Col,FormGroup, Label, Input, Button, Form} from 'reactstrap';
+import '../components/RegisterStyles.css';
+import { validateEmail, validateUsername } from './Validators.js';
+import { getCookie } from './cookie.js';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      usernameValue: "",
+      emailValue: "",
+      passwordValue: "",
+      validForm: false,
+      tokenValue : "",
+    }
+    this.Register = this.Register.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.Validate = this.Validate.bind(this);
+
+  }
+  handleChange({ target }) {
+    this.setState({
+      [target.name]: target.value,
+    });
+    let isValid = true;
+    let stateUpdate = {
+      invalidPassword: this.state.invalidPassword,
+      invalidEmail: this.state.invalidEmail,
+      invalidUsername: this.state.invalidUsername
+     }
+    if (target.name === "passwordValue" ) {
+      stateUpdate.invalidPassword= false;
+    };
+    if (target.name === "emailValue" && !validateEmail(this.state.emailValue)) {
```

Comment:
These can all be else if { statements.

Reply:
If they are else statements then not all the errors receive feedback 
@grootendorste